deployment/ubuntu-package-armhf/Dockerfile.amd64

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:bionic
+# Docker build arguments
+ARG SOURCE_DIR=/jellyfin
+ARG PLATFORM_DIR=/jellyfin/deployment/ubuntu-package-armhf
+ARG ARTIFACT_DIR=/dist
+ARG SDK_VERSION=2.2
```

Comment:
Aren't we using 3.0 for arm? Or is it for some arms only?

Reply:
Not for the Debian packages that seem to work fine.